src/runtime/job.cpp

```diff
@@ -861,6 +863,7 @@ static void launch(JobTable *jobtable) {
     sigprocmask(SIG_UNBLOCK, &set, 0);
     pid_t pid = wake_spawn(cmdline[0], cmdline, environ);
     sigprocmask(SIG_BLOCK, &set, 0);
+    wcl::log::info("Spawned job: label = '%s', pid = %d", job_label_str->c_str(), pid)();
```

Comment:
This will cause log sizes to grow super fast. I'm a bit concerned about users having huge `wake.log`s that they don't think to delete

Reply:
Ok I'll remove it for now then, if I still want it I can add in a flag for it.

Reply:
I can get this info from the database so its still there, I just have to find it.